lib/CL/pocl_cache.c

```diff
@@ -563,12 +563,12 @@ pocl_cache_init_topdir ()
       return CL_FAILED;
 
 #elif defined(_WIN32)
-      tmp_path = getenv ("TEMP");
+      tmp_path = pocl_get_string_option ("LOCALAPPDATA");
       if (!tmp_path) {
```

Comment:
Yes, this is existing code, but if there are other changes to this PR, please move the curly bracket to the next line as in the GNU C style.